cognite/client/_api/extractionpipelines.py

```diff
@@ -117,22 +118,26 @@ def list(self, limit: int = 25) -> ExtractionPipelineList:
         return self._list(list_cls=ExtractionPipelineList, resource_cls=ExtractionPipeline, method="GET", limit=limit)
 
     @overload
-    def create(self, extractionPipeline: ExtractionPipeline) -> ExtractionPipeline:
+    def create(self, extraction_pipeline: ExtractionPipeline, **kwargs: Dict[str, Any]) -> ExtractionPipeline:
         ...
 
     @overload
-    def create(self, extractionPipeline: Sequence[ExtractionPipeline]) -> ExtractionPipelineList:
+    def create(
+        self, extraction_pipeline: Sequence[ExtractionPipeline], **kwargs: Dict[str, Any]
+    ) -> ExtractionPipelineList:
         ...
 
     def create(
-        self, extractionPipeline: Union[ExtractionPipeline, Sequence[ExtractionPipeline]]
+        self,
+        extraction_pipeline: Union[ExtractionPipeline, Sequence[ExtractionPipeline]] = None,
+        **kwargs: Dict[str, Any],
```

Comment:
Not a big fan of using `**kwargs` for this as it will silently swallow any additional named arguments. Why not this?
```
extraction_pipeline: Union[ExtractionPipeline, Sequence[ExtractionPipeline]] = None,
extractionPipeline: Union[ExtractionPipeline, Sequence[ExtractionPipeline]] = None
```

Reply:
Agreed! …which is why I added `handle_deprecated_camel_case_argument`

Reply:
(It raises and warns depending on usage and is reused for in all functions with deprecated parameter names 😄 )

Reply:
Ah, I missed the fact that it raises if it receives unexpected kwargs. Great! Let's add some unit tests for that function though.

Reply:
Ah, yes, on it! 🙈